### PR TITLE
Add support for running commands

### PR DESCRIPTION
**Pull request by avranju**
Partially addresses #3. Adds the ability to run commands under feature `write`. Fixes some minor clippy warnings and doctest errors.

### CODE REVIEW

File: rups/src/cmd.rs [hunk 1]
```diff
@@ -44,6 +49,8 @@ impl<'a> Command<'a> {
             Self::SetUsername(username) => vec![username],
             Self::SetPassword(password) => vec![password],
             Self::List(query) => query.to_vec(),
+            #[cfg(feature = "write")]
+            Self::Run(cmd, param) => vec![cmd, param.unwrap_or("")],
```
Comment (aramperes) [inline rups/src/cmd.rs]: It should emit a Vec of length 1 if there is no param; something like

```suggestion
            Self::Run(cmd, param) => param.map(|param| vec![cmd, param]).unwrap_or_else(|| vec![cmd]),
```

**Review by aramperes — COMMENTED** [review]
Thanks for your pull-request! I just have one request.

File: rups/src/cmd.rs (hunk 1 above)
Comment (avranju) [reply rups/src/cmd.rs]: Makes sense. Made the change.

Comment (avranju) [reply rups/src/cmd.rs]: Ha. Messed up the commit message :). I put some words in `back-tics` and the shell evaluated it. 🤦 Hopefully you can squash the commits when merging!

**Comment by aramperes** [comment]
Thanks again! I'm going to give this library a bit more love and try to cut a release soon.

**Comment by avranju** [comment]
Thanks for this library by the way. I was just about to roll one of my own when I came across yours and found it to be high quality. So decided to contribute to it instead. Thanks again!

**Pull request merged**